tests/ImageSharp.Tests/Formats/Png/PngSmokeTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿// <copyright file="PngSmokeTests.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Tests.Formats.Png
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.IO;
+    using Xunit;
+    using ImageSharp.Formats;
+    using System.Linq;
+    using ImageSharp.IO;
+
+    public class PngSmokeTests
+    {
+        [Theory]
+        [WithTestPatternImages(300, 300, PixelTypes.All)]
+        public void GeneralTest<TColor>(TestImageProvider<TColor> provider)
+            where TColor : struct, IPixel<TColor>
+        {
+            // does saving a file then repoening mean both files are identical???
```

Comment:
In the case of Jpeg-s we can apply a higher percentage threshold with `ImageComparer.VisualComparer()`.